packages/tpetra/core/test/MultiVector/MultiVector_MicroBenchmark.cpp

```diff
@@ -66,8 +66,8 @@ namespace { // (anonymous)
     typedef Tpetra::MultiVector<SC, LO, GO, NT> MV;
     typedef Teuchos::ScalarTraits<SC> STS;
     
-    SC ONE = STS::one();
-    SC ZERO = STS::zero();
+    typename MV::impl_scalar_type ONE = STS::one();
```

Comment:
I didn't write this code.  On the other hand, `putScalar` takes `const SC&`, and `STS` is `Teuchos::ScalarTraits<SC>`, so this code should build.  Nevertheless, I'm OK with this change.